linchpin/provision/library/os_server2.py

```diff
@@ -18,20 +18,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this software.  If not, see <http://www.gnu.org/licenses/>.
 
-# flake8: noqa
```

Comment:
You removed this line. flake8 fails because you are NOT ignoring this file. Please put it back and flake8 will not evaluate this file.